superset/migrations/shared/migrate_viz/processors.py

```diff
@@ -131,3 +133,59 @@ class MigrateSunburst(MigrateViz):
     source_viz_type = "sunburst"
     target_viz_type = "sunburst_v2"
     rename_keys = {"groupby": "columns"}
+
+
+class TimeseriesChart(MigrateViz):
+    has_x_axis_control = True
+
+    def _pre_action(self) -> None:
+        self.data["contributionMode"] = "row" if self.data.get("contribution") else None
```

Comment:
Is this (and other attributes) required? Generally if not I think it's cleaner/safer to leave these undefined and fallback to the default.

Reply:
These rules were added to keep visual consistency between the previous and new versions.